examples/angular-cli/.storybook/webpack.config.ts

```diff
@@ -1,6 +1,6 @@
-const path = require('path');
+import { resolve } from 'path';
```

Comment:
mixing es modules & commonjs here, is that recommended?

Reply:
Is it not recommended? A common linting rule in `ts` is `no-use-require` or something like that. I can revert that if you want, it works though. I never expected any trouble using imports for path like that

Reply:
https://github.com/webpack/webpack/issues/4039#issuecomment-273804003
what about this?

Reply:
This post is 2 years old and maybe babel / typescript already converts my import to es5 (can't check right now)

Reply:
I came across this issue yesterday itself where I was mixing the two. This led to me getting some imports inside `default` and some not. Not mixing them up helped me fix it. Although I am not quite sure about your usage.